src/resources/helm_chart.rs

```diff
@@ -37,6 +37,16 @@ impl HelmChart {
             None => bail!("No actions schema for helm chart"),
         }
     }
+
+    pub fn get_actions(&self) -> Result<Vec<platz_chart_ext::ChartExtActionV0>> {
```

Comment:
It would be easier to use `platz_chart_ext::ChartExtActions` as with `ChartExtFeatures`

Reply:
I don't understand what you mean. Can you please explain?

Reply:
Instead of defining `pub actions_schema: Option<serde_json::Value>` (line 17) it could be `pub actions_schema: Option<platz_chart_ext::ChartExtActions>`, saving the parsing code below.

Reply:
Fixed